QModManager/Patching/Patcher.cs

```diff
@@ -134,14 +169,24 @@ internal static void Patch()
                 Logger.Fatal($"A fatal patching exception has been caught! Patching ended prematurely!");
                 Logger.Exception(pEx);
 
-                Dialog.Show("A fatal patching exception has been caught. QModManager could not be initialized.", Dialog.Button.close, Dialog.Button.Disabled, false);
+                new Dialog()
+                {
+                    message = "A fatal patching exception has been caught. QModManager could not be initialized.",
+                    color = Dialog.DialogColor.Red,
+                    leftButton = Dialog.Button.SeeLog,
+                }.Show();
             }
             catch (Exception e)
             {
-                Logger.Fatal("An unhandled exception has been caught! - Patching ended prematurely!");
+                Logger.Fatal("An unhandled exception has been caught! Patching ended prematurely!");
                 Logger.Exception(e);
 
-                Dialog.Show("An unhandled exception has been caught. QModManager could not be initialized.", Dialog.Button.close, Dialog.Button.Disabled, false);
+                new Dialog()
+                {
+                    message = "An unhandled exception has been caught. QModManager could not be initialized.",
+                    color = Dialog.DialogColor.Red,
+                    leftButton = Dialog.Button.SeeLog,
+                }.Show();
```

Comment:
Not an issue but a reminder for myself:  
The `Patcher` class is starting to get a bit long.

Find opportunities to further break down its many steps into isolated methods or classes that can make the overall orchestration easier to follow.

Reply:
If we split it up further, the repo will become a mess. I would suggest leaving the `Patcher` class as it is now.

(It's less than 200 lines. Remember when it was 600?)